docker/datahub-ingestion-base/Dockerfile

```diff
@@ -31,15 +32,30 @@ RUN apt-get update && apt-get install -y \
     zip \
     unzip \
     ldap-utils \
-    openjdk-11-jre-headless \
-    && python -m pip install --upgrade pip wheel setuptools==57.5.0 \
+    && python -m pip install --no-cache --upgrade pip wheel setuptools==57.5.0 \
     && curl -Lk -o /root/librdkafka-${LIBRDKAFKA_VERSION}.tar.gz https://github.com/edenhill/librdkafka/archive/v${LIBRDKAFKA_VERSION}.tar.gz \
     &&  tar -xzf /root/librdkafka-${LIBRDKAFKA_VERSION}.tar.gz -C /root \
     &&  cd /root/librdkafka-${LIBRDKAFKA_VERSION} \
-    &&  ./configure --prefix /usr && make && make install && make clean && ./configure --clean \
-    && apt-get remove -y make
+    &&  ./configure --prefix /usr && make && make install && make clean && ./configure --clean
+
 COPY --from=binary /go/bin/dockerize /usr/local/bin
 
+COPY ./docker/datahub-ingestion-base/base-requirements.txt requirements.txt
+COPY ./docker/datahub-ingestion-base/entrypoint.sh /entrypoint.sh
+
+RUN pip install --no-cache -r requirements.txt && \
+    pip uninstall -y acryl-datahub && \
+    chmod +x /entrypoint.sh && \
+    addgroup --gid 1000 datahub && \
+    adduser --disabled-password --uid 1000 --gid 1000 --home /datahub-ingestion datahub
+
+ENTRYPOINT [ "/entrypoint.sh" ]
+
+FROM ${BASE_IMAGE} as full-install
+
+RUN apt-get install -y -qq \
```

Comment:
does this not need a `apt-get update`? normally best practice is to run apt-get update at the start of every RUN command, and `rm -rf /var/lib/apt/lists/*` at the end

if you don't do it on every RUN command individually, it doesn't actually reduce the size of the image because they're in separate layers

I think it's the same situation with removing `make`